src/core/store/modules/lines/lines.reducer.ts

```diff
@@ -32,12 +33,14 @@ export const linesInitialState: CreditLineState = {
   user: {
     linePositions: {},
     lineAllowances: {},
+    borrowerPositions: {},
```

Comment:
if we make `user.linePositions` of type `UserPositionSummary` (which includes current CreditPosition + metadata on role e.g. 'borrower) then we dont need separate `borrowerPosition` we can just get it by  `user.linePositions.filter(p => p.role == BORROWER_POSITION_ROLE)` in a selector